token/bearer.go

```diff
@@ -90,19 +118,24 @@ func (b *BearerToken) SetOwner(id *owner.ID) {
 	b.token.SetBody(body)
 }
 
-func (b BearerToken) OwnerID() *owner.ID {
-	return owner.NewIDFromV2(b.token.GetBody().GetOwnerID())
+// OwnerID returns owner ID value of the user who can attach bearer token to
+// its requests.
+func (b BearerToken) OwnerID() owner.ID {
+	return *owner.NewIDFromV2(b.token.GetBody().GetOwnerID())
 }
 
-func (b *BearerToken) SignToken(key *ecdsa.PrivateKey) error {
-	err := sanityCheck(b)
+// SignToken signs bearer token. This method should be invoked with the private
```

Comment:
Could you pls remind me if different token signature schemes are supported?

Reply:
Yes, they are.